akka-cluster/src/test/scala/akka/cluster/ClusterSpec.scala

```diff
@@ -65,6 +65,11 @@ class ClusterSpec extends AkkaSpec(ClusterSpec.config) with ImplicitSender {
       info.getOperations.length should be > (0)
     }
 
+    "reply with InitJoinNack for InitJoin before joining" in {
+      system.actorSelection("/system/cluster/core/daemon") ! InitJoin(system.settings.config)
```

Comment:
Could this ever be a large remote message, with `config`? I know it's been in use for a while.

Reply:
In real implementation it's not sending the full config, only the properties defined by the join config checkers.